tests/struct.c

```diff
@@ -0,0 +1,50 @@
+/* Ordinary struct */
+struct coordinates {
+    int x,y;
+};
+
+/* Struct with enum declaration inside */
+struct rainbow {
+    enum Fruit{Violet, Indigo, Blue, Green, Yellow, Red}field1;
+};
+
+/* Anonymous struct */
+struct {
```

Comment:
This is not valid in C99, right?
Add a variable declaration instead:
```suggestion
struct {
   char alpha;
   int num;
} a;
```